plugin/web/extensions/sidebar/modals/datacatalog/components/content/DatasetsPage/DatasetTree/Folder.tsx

```diff
@@ -88,7 +88,6 @@ const Wrapper = styled.div<{ isOpen?: boolean }>`
     isOpen
       ? "height: 100%;"
       : `
-  height: 29px; 
```

Comment:
This and the other height should be kept, but converted to min-height. Without this, the Folders that don't have long names become much shorter than before